backend/app/services/adapters/task_kinds.py

```
@@ -95,23 +95,29 @@ def create_task_or_append(
                     status_code=400,
                     detail="task already clear, please create a new task",
                 )
-
-            # Check if task is expired
             expire_hours = settings.APPEND_CHAT_TASK_EXPIRE_HOURS
+            # Check if task is expired
             task_type = (
                 task_crd.metadata.labels
                 and task_crd.metadata.labels.get("taskType")
```

Ruff (0.14.7)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)